src/scripts/panorama.json

```
@@ -6,5 +6,12 @@
     "color": "white",
     "background_color": "#1a2735"
   },
-  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#panorama"
+  "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#panorama",
+  "preferences": {
+    "maxPostWidth": {
+      "label": "Max. post width",
+      "type": "text",
+      "default": "854px"
```

Really weird-looking default, is this a typo? For fun I calculated what the maximum space for posts would be for 1080p (1485px, a little big for Tumblr posts) and for 720p... it's 845px :D

```suggestion
      "default": "845px"
```

---

a) I believe it is not, but b) I have no recollection whatsoever about where I came up with that figure! "a 3-wide photoset displays a 540px image at exactly 1:1 on a retina display" is 818, so that's not it.

---

(also, isn't that 794px? not that it matters; this is all so arbitrary :D)

---

...It's actually 830px, I forgot about the 16px total outer padding on `.container`, and the fact the sidebar shrinks to 319px if possible because the dimensions are done wrong.

Sidebar width = ~~320px~~ 319px
Sidebar margin = 30px
Floating avatar gutter = 85px
Container padding = ~~0px~~ 16px

990px (desktop breakpoint) - 319 - 30 - 85 - 16 = 540px
1280px (720p X dimension) - 319 - 30 - 85 - 16 = 830px

---

Ah, right: Panorama overrides the container padding to be... well, okay, I don't actually know why it's 30px on the right side; logically it should be 20px so the header is unmodified.

---

**current, increased container padding**

Sidebar width = 320px
Sidebar margin = 30px
Floating avatar gutter = 85px
Container padding = 21px (to center the avatar) left + 30px (to center the sidebar) right

990px (desktop breakpoint) - 320 - 30 - 85 - 21 - 30 = 504px (too small)
1280px (720p X dimension) - 320 - 30 - 85 - 21 - 30 = 794px

**8px container padding**

Sidebar width = 320px
Sidebar margin = 30px
Floating avatar gutter = 85px
Container padding = 8px + 8px

990px (desktop breakpoint) - 320 - 30 - 85 - 16 = 539px (still too small because the magic 319 doesn't trigger whee)
1280px (720p X dimension) - 320 - 30 - 85 - 16 = 829px